### PR TITLE
feat: add drag and drop wizard

### PR DESCRIPTION
**Pull request by ruisaraiva19**
re #30

### CODE REVIEW

File: components/drag-and-drop/index.tsx [hunk 1]
```diff
@@ -1,46 +1,364 @@
...
+								</div>
+								<div className={classnames(styles.preview, styles.rounded)}>
+									<img src={imageBase64} alt="Favicon preview rounded" />
+								</div>
+							</div>
+							<Typography variant="title" weight="bold" className={styles.imagePreviewTitle}>
+								Your new favicon is ready!
+							</Typography>
+							<Typography variant="footer" weight="semiBold" color="gray" className={styles.imagePreviewSubtitle}>
+								We’ve prepared a zip file with all the sizes and a README with the code plus other resources.
```
Comment (rgllm) [inline components/drag-and-drop/index.tsx:299]: Instead of code we can say meta tags here

File: components/drag-and-drop/index.tsx [hunk 2]
```diff
@@ -1,46 +1,364 @@
...
+							closeTimeoutMS={200}
+							contentLabel="Code generated">
+							<div className={styles.modalContainer}>
+								<div className={styles.modalHeader}>
+									<Typography variant="title" weight="bold">
+										Insert the following code in the &lt;head&gt; section of your pages:
+									</Typography>
+									<Clipboard component="div" data-clipboard-text={headTemplate(undefined, isSvg, pwa)}>
+										<Button color="white" background="bgLink">
+											Copy code
```
Comment (rgllm) [inline components/drag-and-drop/index.tsx:350]: Same here

File: components/drag-and-drop/index.tsx (hunk 1 above)
Comment (ruisaraiva19) [reply components/drag-and-drop/index.tsx:299]: @otsugua 👀

File: components/drag-and-drop/index.tsx (hunk 2 above)
Comment (ruisaraiva19) [reply components/drag-and-drop/index.tsx:350]: @otsugua 👀

**Pull request merged**